src/operations/finances.md

```diff
@@ -0,0 +1,67 @@
+# Finances
+
+The purpose of this document is to describe the financial status and processes of the Toronto Community Network.
+
+## Introduction
+The Toronto Community Network is a community project that current does not have its own independent legal entity or bank accounts, and relies on its fiscal hosting partners to provide fiscal administration capacity.
+Aside from out-of-pocket spendings from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).
+
+## Internet Society Grant
+TBA (some information about the grant)
+
+### Administration
+Free Geek Toronto currently holds the full amount of the grant, and the balance of X (after 10% of administrative fees) is to be distributed for the project according to the budget set out in the grant proposal.
```

Comment:
Possibly include the two scheduled deposits as well @TimTor

Reply:
```suggestion
Instead of "according to the budget set out in the grant proposal." suggest "according to the Memorandum of Understanding with Free Geek Toronto." since this doc is about the relationsship with FGT rather than ISOC
```